<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tomo Slicer 0.3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="tomo::ToolchainEditor" name="toolchainEditor" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="tomo::widget::ToolchainGL" name="toolchainVisualizer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockNavigator">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Navigator</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="tomo::ToolchainNavigator" name="navigator"/>
  </widget>
  <widget class="QDockWidget" name="dockLibrary">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="tomo::ToolLibrary" name="toolLibrary"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionDelete_selection"/>
   </widget>
   <widget class="QMenu" name="menuToolchain">
    <property name="title">
     <string>Toolchain</string>
    </property>
    <addaction name="actionProcess"/>
    <addaction name="actionLoad_From_Macro"/>
    <addaction name="actionSave_As_Macro"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuToolchain"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockParameters">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="tomo::widget::ElementForm" name="parameters"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionDelete_selection">
   <property name="text">
    <string>Delete selection</string>
   </property>
  </action>
  <action name="actionMove_to_parent">
   <property name="text">
    <string>Move to parent</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="text">
    <string>Process</string>
   </property>
  </action>
  <action name="actionSave_As_Macro">
   <property name="text">
    <string>Save As Macro...</string>
   </property>
  </action>
  <action name="actionLoad_From_Macro">
   <property name="text">
    <string>Load From Macro...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomo::widget::ToolchainGL</class>
   <extends>QWidget</extends>
   <header>widget/ToolchainGL.h</header>
   <container>1</container>
   <slots>
    <slot>light2cam()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>tomo::ToolchainNavigator</class>
   <extends>QWidget</extends>
   <header>widget/ToolchainNavigator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomo::ToolLibrary</class>
   <extends>QWidget</extends>
   <header>widget/ToolLibrary.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomo::ToolchainEditor</class>
   <extends>QWidget</extends>
   <header>widget/ToolchainEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomo::widget::ElementForm</class>
   <extends>QWidget</extends>
   <header>widget/ElementForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
